src/config/nodemailer.ts

```diff
@@ -0,0 +1,28 @@
+const nodemailer = require("nodemailer");
+import Mail from "nodemailer/lib/mailer";
+
+const createGmailTransporter = () => {
+  const hostname = process.env.SMTP_SERVER;
+  const port = Number(process.env.SMTP_PORT) as number;
```

Comment:
`const port = process.env.SMTP_PORT as number;`
I do not consider double parsing as a great idea, that would be an unnecessary idea.